css/style.css

```diff
@@ -376,8 +376,6 @@ input:focus:invalid {
   box-shadow: none;
 }
 
```

Comment:
### Your Mobile Menu
![menu-student](https://user-images.githubusercontent.com/93039359/196632499-f6ff449f-fe1e-4962-b28a-96488f2c5d31.PNG)

### Figma Mobile Menu
![menu-figma](https://user-images.githubusercontent.com/93039359/196632530-742c7bdf-a8d6-4dcc-80a7-4bba92ae9b76.PNG)

- [ ] Your mobile menu should have the same background-color as you can see in the Figma template to make your design look consistent with your template 👍 
- [ ] Your menu should blur everything behind your menu as you can see in the Figma template to make your design more professional and human friendly 😊